src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/ContributorTeamRepository.cs

```diff
@@ -93,7 +100,7 @@ public void Add(ContributorTeam newEntity)
         /// <param name="updatedEntity">Updated contributor team.</param>
         public void Update(ContributorTeam updatedEntity)
         {
-            throw new NotImplementedException();
+            _unitOfWork.Context.ContributorTeams.Single(t => t.Id == updatedEntity.Id);
```

Comment:
Same thing

Reply:
implementing this method instead of throwing exception. Revert changes?  same thing